src/order/category/DistribLattice.lean

```diff
@@ -0,0 +1,59 @@
+/-
+Copyright (c) 2022 Yaël Dillies. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yaël Dillies
+-/
+import order.category.Lattice
+
+/-!
+# The category of distributive lattices
+
+This file defines `DistribLattice`, the category of distributive lattices.
```

Comment:
Perhaps worth commenting that the lattices here don't need to be bounded, which is sometimes taken as the definition of DistLat (https://ncatlab.org/nlab/show/DistLat), similarly the homs don't need to preserve top/bottom

Reply:
"Sometimes"? I would think it would be much closer to always?

Reply:
Perhaps! I don't have enough evidence to be sure either way, please feel free to overrule.